### PR TITLE
Split /posts/favourites and /posts

### PR DESCRIPTION
**Pull request by awildbrysen**
See #129

### CODE REVIEW

File: api/src/main/kotlin/com/gmtkgamejam/routing/PostRoutes.kt [hunk 1]
```diff
@@ -23,87 +22,78 @@ fun Application.configurePostRouting() {
     val service = PostService()
     val favouritesService = FavouritesService()
 
+    fun getFilterFromParameters(params: Parameters): List<Bson> {
+        val filters = mutableListOf<Bson>(/*PostItem::deletedAt eq null*/)
```
Comment (Willdotwhite) [inline api/src/main/kotlin/com/gmtkgamejam/routing/PostRoutes.kt:26]: If `PostItem::deletedAt eq null` is commented out, this query will bring back deleted posts

Comment (awildbrysen) [reply api/src/main/kotlin/com/gmtkgamejam/routing/PostRoutes.kt:26]: Oh forgot to revert after testing it out 🙈

**Review by Willdotwhite — APPROVED** [review]

**Comment by Willdotwhite** [comment]
@awildbrysen want me to merge this in whenever?

**Comment by Willdotwhite** [comment]
Let's all ignore what happened on this branch today, and pretend it never happened

**Pull request merged**